spec/forms/create_group_assignments_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe CreateGroupAssignments, '#call' do
+  let(:users) { create_list(:guider, 2) }
+  let(:user_ids) { users.map(&:id) }
+  let(:group_params) { Hash[name: 'Another Group'] }
+
+  subject { described_class.new(user_ids, group_params) }
+
+  context 'when the users are not already members' do
+    it 'assigns the group to the given users' do
+      # ensure just the one group is created
+      expect { subject.call }.to change { Group.count }.by(1)
+
+      users.each do |user|
+        expect(user.groups.map(&:name)).to eq(['Another Group'])
+      end
```

Comment:
You could definitely argue that. I collapsed them though as the one logical assertion per spec gets stupid-slow after a while 😢